graylog2-server/src/main/java/org/graylog2/rest/resources/entities/preferences/model/EntityListPreferences.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog2.rest.resources.entities.preferences.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public record EntityListPreferences(@JsonProperty("displayed_attributes") List<String> displayedAttributes,
+                                    @JsonProperty("sort") String sort,
```

Comment:
Maybe some enums for sort and order?

Reply:
BE will never do anything with this value, just store and retrieve. Won't enums on FE side be enough?